src/fu-util.c

```diff
@@ -2557,6 +2745,24 @@ main (int argc, char *argv[])
 		     /* TRANSLATORS: command description */
 		     _("Gets the host security attributes."),
 		     fu_util_security);
+	fu_util_cmd_array_add (cmd_array,
+		     "block-firmware",
+		     "[CHECKSUM]",
+		     /* TRANSLATORS: command description */
+		     _("Blocks a specific firmware from being installed."),
+		     fu_util_block_firmware);
+	fu_util_cmd_array_add (cmd_array,
+		     "unblock-firmware",
+		     "[CHECKSUM]",
+		     /* TRANSLATORS: command description */
+		     _("Blocks a specific firmware from being installed."),
+		     fu_util_unblock_firmware);
+	fu_util_cmd_array_add (cmd_array,
+		     "get-blocked-firmware",
+		     NULL,
+		     /* TRANSLATORS: command description */
+		     _("Gets the list of blocked firmware."),
+		     fu_util_get_blocked_firmware);
```

Comment:
This same set of commands should be added to fu-tool.c too

Reply:
I'm not so sure about this; the `approved` stuff isn't in `fwupdtool` either, and I don't think it's particularly useful to allow the admin running as root to say "don't install foo" which then prevents the (same?) admin from installing that firmware update. I don't think fwupdtool has to have the same featureset as fwupdmgras longer term I want to push the power users using fwupdtool to use the daemon even if the daemon auto-quits straight after it's completed.